docs/controls/TabbedCommandBar.md

```diff
@@ -0,0 +1,209 @@
+            </ComboBox>
+        </AppBarElementContainer>
+        <AppBarToggleButton Icon="Bold" Label="Bold" />
+        <AppBarToggleButton Icon="Italic" Label="Italic" />
+        <AppBarToggleButton Icon="Underline" Label="Underline" />
+    </ribbon:TabbedCommandBarItem>
+
+    <ribbon:TabbedCommandBarItem Header="Insert">
+        <AppBarButton Icon="Pictures" Label="Pictures">
+            <AppBarButton.Flyout>
+                <MenuFlyout Placement="BottomEdgeAlignedLeft">
+                    <MenuFlyoutItem Text="This Device">
+                        <MenuFlyoutItem.Icon>
+                            <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xEC4E;" />
+                        </MenuFlyoutItem.Icon>
+                    </MenuFlyoutItem>
+                    <MenuFlyoutItem Text="Stock Images">
+                        <MenuFlyoutItem.Icon>
+                            <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xE721;" />
+                        </MenuFlyoutItem.Icon>
+                    </MenuFlyoutItem>
+                    <MenuFlyoutItem Icon="Globe" Text="Online Pictures" />
+                </MenuFlyout>
+            </AppBarButton.Flyout>
+        </AppBarButton>
+
+        <ribbon:TabbedCommandBarItem.SecondaryCommands>
+            <AppBarButton Icon="Add" Label="New item" />
+        </ribbon:TabbedCommandBarItem.SecondaryCommands>
+    </ribbon:TabbedCommandBarItem>
+</ribbon:TabbedCommandBar>
+```
+
+## Sample Output
+
+ ![TabbedCommandBar Overview](../resources/images/Controls/TabbedCommandBar/Overview.gif)
+
+## Properties
+
+### TabView Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| CanCloseTabs | bool | Default value for if the TabViewItem doesn't specify a IsClosable value. |
+| IsCloseButtonOverlay | bool | Changes the behavior of how the Close button effects layout.  If True, the close button will overlay itself on top of content if the tab is a fixed size. |
+| ItemHeaderTemplate | DataTemplate | Default Template for the TabViewItem Header if no template is specified. |
+| SelectedTabWidth | double | Size to set the Selected Tab Header.  Defaults to Auto. |
+| TabActionHeader | object | Area to the Right of Tab Strip before Padding. |
+| TabActionHeaderTemplate | DataTemplate | Template for the TabActionHeader. |
+| TabEndHeader | object | Area to the Right of the Tab Strip after Padding. |
+| TabEndHeaderTemplate | DataTemplate | Template for the TabEndHeader. |
+| TabStartHeader | object | Description Area to the Left of the Tab Strip. |
+| TabStartHeaderTemplate | DataTemplate | Template for the TabStartHeader. |
+| TabWidthBehavior | TabWidthMode | Actual, Equal, or Compact values specify how Tab Headers should be sized.  Defaults to Actual. |
+
+> [!IMPORTANT]
+> Do not use `ItemsStackPanel` if you override the ItemsPanel.  It is suggested to keep the `TabWidthBehavior` to `Actual` when using a custom panel.
```

Comment:
If you are going to discourage using ItemsStackPanel, make a better recommendation at the same time.